pipettes/core/tasks.cpp

```diff
@@ -113,7 +114,7 @@ void pipettes_tasks::start_tasks(
     spi_task_client.set_queue(&spi_task.get_queue());
 
     auto& eeprom_task =
-        eeprom_task_builder.start(5, "eeprom", i2c3_task_client, queues);
+        eeprom_task_builder.start(5, "eeprom", i2c3_task_client, eeprom_wp_pin);
```

Comment:
Mistake? Don't we still need to give the queue client to the eeprom task builder?

Reply:
We do not. The eeprom task only communicates with an i2c task.